master/internal/user/postgres_long_lived_tokens_intg_test.go

```diff
@@ -0,0 +1,96 @@
+//go:build integration
+// +build integration
+
+package user
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/o1egl/paseto"
+	"github.com/stretchr/testify/require"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+func getLongLivedTokensEntry(ctx context.Context, userID model.UserID) (bool, error) {
+	return db.Bun().NewSelect().Table("long_lived_tokens").
+		Where("user_id = ?", userID).Exists(ctx)
+}
+
+func TestDeleteAndCreateLongLivedToken(t *testing.T) {
+	user, err := addTestUser(nil)
+	require.NoError(t, err)
+
+	// Add a LongLivedToken.
+	token, err := DeleteAndCreateLongLivedToken(context.TODO(), user.ID)
+	require.NoError(t, err)
+	require.NotNil(t, token)
+
+	exists, err := getLongLivedTokensEntry(context.TODO(), user.ID)
+	require.True(t, exists)
+	require.NoError(t, err)
+}
+
+func TestDeleteAndCreateLongLivedTokenHasExpiresAt(t *testing.T) {
+	user, err := addTestUser(nil)
+	require.NoError(t, err)
+
+	// Add a LongLivedToken with custom (Now() + 3 Months) Expiry Time.
+	expiresAt := time.Now().Add(TokenExpirationDuration * 3)
```

Comment:
Is this another case where UTC should be used?

Reply:
Corrected it